src/MongoDB.Driver/Encryption/MongocryptdFactory.cs

```diff
@@ -149,6 +148,16 @@ private bool ShouldMongocryptdBeSpawned(out string path, out string args)
                 {
                     args += " --idleShutdownTimeoutSecs 60";
                 }
+
+                if (!args.Contains("logpath")) // disable logging by the mongocryptd process
+                {
+                    args += " --logpath nul";
```

Comment:
Do we need a comment that this is platform specific and we will need to use different magic values (e.g. `/dev/null`) on other platforms?

Reply:
Agreed. Done